.github/workflows/build.yml

```diff
@@ -0,0 +1,19 @@
+name: build
+on:
+  pull_request:
+    paths:
+    - '**'
+    - '!doc/contributors/**'
+    - '!doc/design/**'
```

Comment:
This still needs to be updated to handle the periodic testing case if I'm reading the following section in (the now deleted) test-scripts.yml workflow definition:

```
  schedule:
  - cron: '0 0 * * *' # daily to pick up releases
```

/hold